.storybook/preview-body.html

```diff
@@ -1,3 +1,3 @@
 <body
-  class="bg-white font-open-sans text-black dark:bg-black dark:text-white"
+  class="bg-white font-open-sans text-neutral-950 dark:bg-neutral-950 dark:text-white"
```

Comment:
@ovflowd I just realized that we have this `preview-body.html`, previously I was searching for `.css` files.